### PR TITLE
Import complicated sys

### PR DESCRIPTION
**Pull request by AndrewJack**
First

### CODE REVIEW

File: 404.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+permalink: /404.html
+layout: default
+published: true
+---
+
+Sorry, we couldn't find that page 🤷.
+
+Try [going back to the latest notes](/), or [searching the site](https://www.google.com/search?hl=en&q=site%3Auncomplicated.systems).
```
Comment (AndrewJack) [inline 404.md]: missed one

File: _includes/highlight.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
+.highlight .hll { background-color: #ffffcc }
```
Comment (sjparkinson) [inline _includes/highlight.scss]: 🔥

File: _includes/styles.scss [hunk 3]
```diff
@@ -0,0 +1,317 @@
+@import "normalize";
+@import "highlight";
```
Comment (sjparkinson) [inline _includes/styles.scss]: 🔥

File: _includes/variables.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+// Text
+$section-headings-color: #007d51;
+$body-text-color: #262a33;
+$body-link-color: #0f5499;
+$blockquote-text-color: #262a33;
+
+// Code
+$code-bg-color: #262a33;
+$code-text-color: #ffffff;
```
Comment (sjparkinson) [inline _includes/variables.scss]: 🔥  for the three lines above.

**Pull request merged**